docs/capture_agent_plugins/examples/galicaster_2_manchester/README.md

```diff
@@ -0,0 +1,91 @@
+Peakaboo Plugins for Galicaster
+===================================
+
+Two plugins that enables two-way communication between An Opencast compatible capture agent running Galicaster and a Peakaboo instance. This plugin leverages the Distributed Data Protocol (DDP) to communicate with the Meteor framework. The Audiostream plugin sets up a small audio server on the capture agent using gstreamer.
+
+Things to note
+--------------
+* See https://github.com/hharnisc/python-ddp on more information for developing using ddp
+* When using SSL with peakaboo the websocket url must use 'wss' rather than 'ws'
+* When using a self signed SSL certificate on peakaboo images will not be Posted
```

Comment:
Presumably this would work if you add the self signed certificate to the machine running Galicaster?